config/default.yml

```diff
@@ -469,9 +469,11 @@ RSpec/MissingExampleGroupArgument:
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/MissingExampleGroupArgument
 
 RSpec/MultipleDescribes:
-  Description: Checks for multiple top-level example groups.
+  Description: Checks for multiple top-level example groups or nested example groups.
   Enabled: true
+  Splitting: false
```

Comment:
When there are two mutually exclusive styles, like in this case, RuboCop is using `EnforcesStyle` option.

Reply:
Thank you for your comment.
We will change the code to use `EnforcedStyle` instead of `Splitting` config option.

Reply:
@pirj 
We changed the code to use `EnforcedStyle`.
Could you please confirm that?